inst/templates/mapping_template_AR6.csv

```diff
@@ -1485,9 +1485,11 @@ idx;variable;unit;piam_variable;piam_unit;piam_factor;piam_spatial;internal_comm
 1373;Investment|Infrastructure|Water|Other;billion US$2010/yr;;;;;;;investment into water-related infrastructure (other than for thermal cooling and irrigation);
 1374;Investment|RnD|Energy Supply;billion US$2010/yr;;;;;;;Investments into research and development,energy supply sector;
 1375;Agricultural Demand;million t DM/yr;Demand|++|Crops;Mt DM/yr;;;;;total demand for food,non-food and feed products (crops and livestock) and bioenergy crops (1st & 2nd generation);M
+1375;Agricultural Demand;million t DM/yr;Demand|Bioenergy|+|Secondary products;Mt DM/yr;;;;;total demand for food,non-food and feed products (crops and livestock) and bioenergy crops (1st & 2nd generation);M
```

Comment:
This is a double accouting because `Demand|Bioenergy|+|Secondary products` are already included in `Demand|++|Crops`.

Reply:
Therefore, `Demand|Bioenergy|+|Secondary products` should not be added here.